### DISCUSSION TITLE
Where can translators update Traditional Chinese localization in current?

### DISCUSSION THREAD

**Discussion by jrthsr700tmax** [AutoDarkMode/Windows-Auto-Night-Mode#392] [Q&A]
As title, I had translated Traditional Chinese localization for "Windows Auto Dark Mode" app yet previously, now I would like to optimize its  Traditional Chinese localization, but I couldn't find out the site or platform to translation the localization. Is there any site or platform can let translator who would likes to translating or make the language better in this project? Thank a lot!

**Comment by Armin2208** [comment]
The location files are now located on [AutoDarkModeConfig/Properties/](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/tree/master/AutoDarkModeConfig/Properties).
There is currently no alternative translation plattform (work in progress)
